adapters/tappx/tappx.go

```diff
@@ -133,6 +132,11 @@ func (a *TappxAdapter) buildEndpointURL(params *openrtb_ext.ExtImpTappx, test in
 
 	tappxHost := "tappx.com"
 	isNewEndpoint, err := regexp.Match(`^(zz|vz)[0-9]{3,}([a-z]{2,3}|test)$`, []byte(params.Endpoint))
+	if err != nil {
+		return "", &errortypes.BadInput{
+			Message: "Unable to match params.Endpoint " + string(params.Endpoint) + "): " + err.Error(),
+		}
+	}
```

Comment:
Can you add a test to cover these lines?

Reply:
TODO(dmitris): add a unit test

Reply:
replied below https://github.com/prebid/prebid-server/pull/3621#issuecomment-2070923090 - not seeing an easy way to cover this, and suggest refactoring in a follow-up PR.

Reply:
@AlexBVolcy - are you ok with leaving it as if for now?

Reply:
Yes you can leave it as is, there are other uses of regexp.Match throughout the code base that also dont cover this error.